Xamarin.Forms.Core/MenuItem.cs

```diff
@@ -17,7 +18,8 @@ public class MenuItem : BaseMenuItem, IMenuItemController
 
 		public static readonly BindableProperty IconProperty = BindableProperty.Create("Icon", typeof(FileImageSource), typeof(MenuItem), default(FileImageSource));
 
-		internal static readonly BindableProperty IsEnabledProperty = BindableProperty.Create("IsEnabled", typeof(bool), typeof(ToolbarItem), true);
+		[EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Actually, there already is a controller. Let me know if the public surface area of that controller is not to your liking.